.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug Report
```

Comment:
Please make all fields except `Relevant log output` required, so that the form can't be submitted without filling those.
Please also add an optional text filed "Additional context" in the end